modules/ROOT/pages/getting-started/index.adoc

```diff
@@ -95,10 +95,30 @@ When the schema is generated, you can then execute queries for `actors` and `mov
 
 == Create an instance of `Neo4jGraphQL`
 
-To create an instance of the Neo4j GraphQL Library, you need a Neo4j driver to connect to your database. 
+To create an instance of the Neo4j GraphQL Library, you need a Neo4j driver to connect to your database.
 
-. For a database located at "bolt://localhost:7687", with the username "neo4j" and the password "password", add the following to the bottom of your `index.js` file:
+=== Using AuraDB
+
+. For an AuraDB database, https://neo4j.com/cloud/platform/aura-graph-database/?ref=docs-graphql[create an instance].
+
+. Make sure to save the generated password and the connection URI provided after the instance is ready and looking similar to this:
 +
+image::neo4j-aura-dashboard.png[width=500]
+
+. Create a new file `.env` with the credentials described in the previous step.
+It should look like this:
++
+[source, config]
+----
+NEO4J_USER=neo4j
+NEO4J_PASSWORD=password
+NEO4J_URI=neo4j+s://0083654f.databases.neo4j.io
+----
+
+=== Using a Neo4j database
+
+For a database located at "bolt://localhost:7687", with the username "neo4j" and the password "password", add the following to the bottom of your `index.js` file:
```

Comment:
May be worth noting that the port is configurable in the database so it might be wrong. Or specifying that 7687 is the default but we recommend double checking?

Reply:
I updated it with a link to the page about ports configuration